src/index.html

```diff
@@ -4,9 +4,414 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a class="logo" href="#">
+        <img
+        class="logo__img"
+        src="images/logo.png"
+        alt="Company Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="samsung" class="nav__link">Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="apple" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="samsung" class="nav__link">Samsung</a>
          </li>
```

Reply:
Check [guide](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) and fix everywhere